### ISSUE TITLE
Create a Slack, gitter or something like it to discuss things related to stent?

### ISSUE THREAD

**Issue by pedronasser** [krasimir/stent#17]
I think would be very nice to have somewhere to share thoughts and meet other people experimenting stent in their project.

@krasimir could you create it for us?

**Comment by krasimir** [comment]
Awesome idea!!! Just created a slack channel https://stentjs.slack.com/

**Comment by krasimir** [comment]
Nope, Gitter's better https://gitter.im/stentjs Slack has this weird sign up mode which I forgot about. So Gitter wins over Slack here.
